src/modules/pua_dialoginfo/dialog_publish.c

```diff
@@ -188,6 +188,8 @@ str* build_dialoginfo(char *state, str *entity, str *peer, str *callid,
 			LM_ERR("while adding child\n");
 			goto error;
 		}
+		/* Testing(POC) - Add a display attribute to the remote/identity tag */
+		xmlNewProp(tag_node, BAD_CAST "display", BAD_CAST "RemoteCallerName");	
```

Comment:
I understand that you want the user part from SIP URI set in identity tag, then practically you have to parse it with parse_uri() function, which fills a structure with a field point to user part. Like:

```
parse_uri(remotetarget->s, remotetarget->len, &puri);
```
Then use `puri.user.s` and `puri.user.len`, which point to the start of user part and give the length. Similar will be for localtarget part.